pkg/daemon/daemon.go

```diff
@@ -22,6 +22,7 @@ import (
 	mcfgclientv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
 	"github.com/pkg/errors"
 	"github.com/vincent-petithory/dataurl"
+	fsnotify "gopkg.in/fsnotify.v1"
```

Comment:
Generally it'd be nice to split up "my editor is running a gofmt-like tool" type commits to be first.

Reply:
ack. Yeah it did stick out a little. I will drop this hunk from commit.